tests/bwc/test_rolling_upgrade.py

```diff
@@ -25,15 +25,39 @@
     UpgradePath('5.7.x', '5.8.x'),
     UpgradePath('5.8.x', '5.9.x'),
     UpgradePath('5.9.x', '5.10.x')
-    # TODO: Add upgrade path from 5.10.x to 6.x once the version compatibility issues are fixed
+)
+
+ROLLING_UPGRADES_V5 = (
+    UpgradePath('5.0.x', '5.1.x'),
+    UpgradePath('5.1.x', '5.2.x'),
+    UpgradePath('5.2.x', '5.3.x'),
+    UpgradePath('5.3.x', '5.4.x'),
+    UpgradePath('5.4.x', '5.5.x'),
+    UpgradePath('5.5.x', '5.6.x'),
+    UpgradePath('5.6.x', '5.7.x'),
+    UpgradePath('5.7.x', '5.8.x'),
+    UpgradePath('5.8.x', '5.9'),
+    UpgradePath('5.9', '5.10'),
```

Comment:
Minor: Let's add TODO to replace 5.9 branch again to the release once 5.9.10 is out

Reply:
Added the todo to the 5.9.10 release card: https://github.com/crate/distribute/issues/751

Reply:
Done, I will also update 5.10 to 5.10.x once we release 5.10.1